server/src/main/java/org/opensearch/action/admin/cluster/remotestore/stats/RemoteStoreStats.java

```diff
@@ -43,8 +43,10 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.startObject()
             .field(Fields.SHARD_ID, remoteSegmentUploadShardStats.shardId)
 
-            .field(Fields.LOCAL_REFRESH_TIMESTAMP, remoteSegmentUploadShardStats.localRefreshTimeMs)
-            .field(Fields.REMOTE_REFRESH_TIMESTAMP, remoteSegmentUploadShardStats.remoteRefreshTimeMs)
+            .field(
+                Fields.REFRESH_TIME_LAG_IN_MILLIS,
+                remoteSegmentUploadShardStats.localRefreshTimeMs - remoteSegmentUploadShardStats.remoteRefreshTimeMs
```

Comment:
can we get rid of these fields localRefreshTimeMs and remoteRefreshTimeMs from the class?

Reply:
Kept it as in future we would be using them. We plan to add the actual epoch timestamps

Reply:
the calculation would still become different as those would wall clock time, but these are calculating delta (using System.nanos()). We will need to change anyways, we can change now.